src/common/ErrorPage/styled.js

```diff
@@ -0,0 +1,70 @@
+import styled from "styled-components";
+import { ReactComponent as DangerIcon } from "../../images/danger.svg";
+
+export const Wrapper = styled.div`
+    max-width: 592px;
+    width: 100%;
+    margin: 195px auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px) {
+        margin-top: 120px;
+    }
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+        margin-top: 50px;
+    }
+`;
+
+export const StyledDangerIcon = styled(DangerIcon)`
+    width: 120px;
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+        width: 80px;
+    }
+`;
+
+export const Title = styled.h1`
+    text-align: center;
+    font-weight: 600;
+    font-size: 35px;
+    line-height: 120%;
+    margin: 38.16px 0 0;
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+        font-size: 18px;
+        margin-top: 0;
+    }
+`;
+
+export const Text = styled.p`
+    text-align: center;
+    margin: 24px 82.73px;
+    padding: 0;
+    font-weight: 500;
+    font-size: 22px;
+    line-height: 130%;
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+        font-size: 12px;
+        margin: 15px 40px;
+    }
+`;
+
+export const Button = styled.button`
```

Comment:
In my view it would be better if it was already a React Router Link navigating to homepage.

Reply:
i thought to make only visual side but i can add this feature

Reply:
It is always best to deliver fully working feature. Atomization of tasks is okay, however if this is not too much of a burden we should try to make it work fully. We should better discuss/describe scope of new tasks in the future. As for the Link implementation it is working properly, however you should not nest `<button>` in `<a>` element. Content model of `<a>`  element is 

> [Transparent](https://html.spec.whatwg.org/multipage/dom.html#transparent), but there must be no [interactive content](https://html.spec.whatwg.org/multipage/dom.html#interactive-content-2) descendant, [a](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element) element descendant, or descendant with the [tabindex](https://html.spec.whatwg.org/multipage/interaction.html#attr-tabindex) attribute specified.

There are two ways you could do this in compliance with HTML specification. 

1. Changing `<Button />` styled-component tag from a to React Router Link element like this 
`export const Button = styled(Link)` and passing `to="/"` in `<Button />` .

2. Completely removing Link element and leaving only button element. Then you could pass a function utilizing useNavigate hook in onClick event.

Leaving this decision up to you. [The a element documentation](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element)